docs/topics/templates/document-attributes.adoc

```diff
@@ -38,7 +38,8 @@ ifdef::mta[]
 :ocp-full: OpenShift Container Platform
 :ocp-short: OpenShift
 :ocp: OCP
-:ocp-version: 4.17
+:ocp-version: 4.18
```

Comment:
I think this line needs to be removed.
and leave only the next line with details on the supported OCP versions.
We test MTA 7.2.1 before releasing on OCP-4.16(released), 4.17(released), and 4.18 (will be released Feb 25, which is after MTA 7.2.1 release - Feb 20), 
So I do not want users to get the notion MTA 7.2.1 should wor only along with 4.18, since 4.18 will not be available when MTA 7.2.1 is released.